.github/workflows/ansible-release.yml

```diff
@@ -0,0 +1,109 @@
+name: ansible-release
+on:
+  workflow_dispatch:
+    inputs:
+      ansible_version:
+        description: 'Release Version. Example : 11.1.0'
+        required: true
+      ansible_major_version:
+        description: 'Exmaple 11'
+        required: true
```

Comment:
Future enhancement: make the workflow determine `ansible_version` from `ansible_major_version`,

Reply:
Maybe vice versa?

Reply:
Oops, that's what I meant in the first place.